docs/content/preview/yugabyte-cloud/cloud-monitor/_index.md

```diff
@@ -43,68 +43,16 @@ Integrate with third-party tools such as Datadog, Grafana Cloud, and Sumo Logic
 | [Metrics export](metrics-export/) | Export metrics to third-party monitoring tools. |
 | [Log export](logging-export/) | Export database PostgreSQL query and audit logs to third-party monitoring tools. |
 
-### Cluster activity
+### Tables, nodes, and cluster activity
 
-View cluster activity, node status, and database properties.
+View cluster tables and their tablets, nodes and their status, and audit cluster activity.
```

Comment:
Can't we write it like this: `View cluster tables, tablets, nodes, their statuses, and audit cluster activity?`